components/ContributePage/GetStarted.jsx

```diff
@@ -0,0 +1,74 @@
+import Link from 'next/link';
+import { Fragment } from 'react';
+
+import {
+  SCREEN_WIDTH_LG,
+  SECTION_BOX_CLASS,
+  SUB_HEADER_CLASS,
+} from 'common-util/classes';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Button } from 'components/ui/button';
+import { CTA_LINK } from './utils';
+
+const steps = [
+  {
+    title: 'Link your X account',
+    description: (
+      <>
+        Go to the{' '}
+        <a href={CTA_LINK} className="text-purple-600">
```

Comment:
since the link text has the "↗", can you please add target="_blank"
or can simply use the ExternalLink component

Reply:
Have added this and added _unoptimised_ to the GetOlas section image so that it is clearer